examples/wordpress/backend.jsonnet

```diff
@@ -0,0 +1,229 @@
+local kube = import "lib/kube.libsonnet";
+
+local labels = {
+  tier: "backend",
+};
+
+local master_labels = labels + {
+  component: "master",
+};
+
+local slave_labels = labels + {
+  component: "slave",
+};
+
+{
+  backend_secret: kube.Secret("mariadb") {
+    metadata+: {
+      labels+: labels,
+    },
+    data_+: {
+      "database_name": "webserver_db",
+      "database_user": "webserver_user",
+      "database_password": "webserver_db_password",
+      "replication_user": "replica_user",
+      "replication_password": "replica_password",
+      "root_user": "root_user",
+      "root_password": "root_password"
+  }},
+
+  backend_master_statefulset: kube.StatefulSet("mariadb-master") {
```

Comment:
Suggestion: When you have "internally structured" objects like this, consider using actual jsonnet hierarchies.

Eg:
```
{
  backend: {
     secret: kube.Secret("mariadb") { ... },
     master: {
       statefulset: kube.StatefulSet("mariadb-master") { ... },
     },
  },
  some_reference: $.backend.master.statefulset,
}
```

Most of the time it doesn't make any difference (just replacing `_` with `.`), but sometimes it allows you to build a whole subsection off (eg) a "mariadb_common" template that gets used for both master+slave, or replace/move a whole subtree, etc.